VMBackup/main/handle.py

```diff
@@ -564,6 +565,17 @@ def enable():
         error_msg = 'Failed to call the daemon'
         exit_with_commit_log(temp_status, temp_result,error_msg, para_parser)
 
+def set_file_permission(file_path, permission):
+    os.system('chmod ' + permission + " " + file_path)
+
+def convert_dos_to_unix(file_path):
+    os.system('dos2unix ' + file_path)
+
+def do_log_rotate(config_file_path):
+    set_file_permission(config_file_path, "0644")
+    convert_dos_to_unix(config_file_path)
+    os.system('logrotate ' + config_file_path)
```

Comment:
there is no exception handling. what if logrotate keeps failing. There is no logging on what was generated on successful completion of logrotate.